### ISSUE TITLE
Compatibility with autounattend.xml

### ISSUE THREAD

**Issue by CortexReaver** [riklus/win10install-onusb#2]
Hello, I've got another question rather then a bug report. I've decided to make an unattended installation of windows using this guide: https://github.com/memstechtips/UnattendedWinstall?tab=readme-ov-file 
My question is, will it work with your script? Or do I need to add some commands in it for Windows installation be able to see the autounattend.xml?

I can try to see it myself if it works "as it is", but I rather ask you first to not waste my time :)

**Comment by riklus** [comment]
From my understanding, it seems that memstechtups' project already suggests a way to _partially_ automate the install using Ventoy. As you suggest some command flags are probably needed to load the "autounattend.xml" file, most likely [here](https://github.com/riklus/win10install-onusb/blob/575a4c21ebc79e6000ffd0095c40c0593c9d1f40/wininstall.bat#L158). Take a look at this [documentation page](https://learn.microsoft.com/en-us/windows-hardware/manufacture/desktop/dism-unattended-servicing-command-line-options?view=windows-10), I think it could help you achieve what you want.

**Comment by CortexReaver** [comment]
Got it, so in my case it wiil be:
```
echo [i] Deploying Windows Image...
dism /Apply-Image /ImageFile:R:\RecoveryImage\install.esd /Apply-Unattend:G:\autounattend.xml /Index:1 /ApplyDir:W:\
```

**Comment by riklus** [comment]
Most likely, you could test it and then make a PR like: "if 'autounattend.xml' is present in the USB then use it in the install section".

**Comment by CortexReaver** [comment]
No, it doesn't work for some reason... I still get windows full of crap after the installation was finished.
